archlinuxcn/shadowsocks-rust-git/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: DuckSoft <realducksoft at gmail dot com>
+# Contributor: rustemb <rustemb at systemli dot org>
+pkgname=shadowsocks-rust-git
+pkgver=1.10.9.r5.g89b329c
+pkgrel=1
+provides=(shadowsocks-rust)
+conflicts=(shadowsocks-rust)
+pkgdesc='Rust port of Shadowsocks'
+arch=('x86_64')
+url='https://github.com/shadowsocks/shadowsocks-rust'
+license=('MIT')
+depends=('openssl')
+makedepends=('cargo' 'libsodium')
```

Comment:
```
==> ERROR: Cannot find the git package needed to handle git sources.
==> ERROR: Build failed, check /var/lib/archbuild/extra-x86_64/imlonghao/build
```

Reply:
oops, I missed that when converting `shadowsocks-rust` to `-git`...
reverting...

Reply:
resolved in https://github.com/archlinuxcn/repo/commit/b1e9033118f9ddf8d70df1606bed76f14f9f5471